externs/olx.js

```diff
@@ -5231,6 +5232,14 @@ olx.source.WMTSOptions.prototype.wrapX;
 
 
 /**
+ * Ignored dimensions for prefix keys
+ * @type {Array.<string>|undefined}
+ * @api
+ */
+olx.source.WMTSOptions.prototype.keyPrefixIgnoredDimensions;
```

Comment:
Are all of `key`, `prefix`, and `dimension` really meaningful in this name?  Is `ignoredDimensions` the wrong name (or too broad)?

I don't have a better suggestion, just saying that the name is potentially confusing/meaningless.